examples/simple-transfer/ckb.ts

```diff
@@ -0,0 +1,11 @@
+import { Indexer, RPC, config } from '@ckb-lumos/lumos';
```

Comment:
a chain config file works for me.

Reply:
Do you think we can define an environment variable somewhere so that we can use that var and set the configurations accordingly? (I feel like doing this can save devs' time and effort to switch between environments) Something like the following:

```
const env = process.env.ENVIRONMENT || 'testnet'
export const lumosConfig = env === 'testnet' ? config.predefined.AGGRON4 as config.Config : devConfig as config.Config;
```